xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java

```diff
@@ -229,8 +229,8 @@ public void beginFormat(Format format, Map<String, String> parameters)
         // If the previous format had parameters and the parameters are different from the current ones then close them
         if (this.previousFormatParameters != null) {
             if (parameters.isEmpty()) {
-                // print("(%%)");
-                // this.previousFormatParameters = null;
+                 print("(%%)");
```

Comment:
You did not understood my previous comment. I never told you put back the code (I'm sure you are breaking something by doing that). What I told you is to modify the code in general to not have an empty block anymore.